docs/faq.de.md

```diff
@@ -96,7 +96,7 @@ Mehr über Entbannungsanträge und Entschuldigungen [kannst du hier lesen](/bans
 Achievements kann man während des Spielens auf Timolia abschließen. Eine Übersicht über alle Achievements findest du unter dem Befehl `/achievements`.
 
 ### Woher bekomme ich das Timolia Optifine-Cape?
-- URL: [https://www.needcoolshoes.com/banner?=paeyeseqpwphec](https://www.needcoolshoes.com/banner?=paeyeseqpwphec)
+- URL: [https://www.needcoolershoes.com/banner?=paeyeseqpwphec](https://www.needcoolshoes.com/banner?=paeyeseqpwphec)
```

Comment:
You forgot to change the URL you are linking to, please also change this one

Reply:
done